-- Found rosidl_default_generators: 1.0.1 (/opt/ros/foxy/share/rosidl_default_generators/cmake)
-- Found ament_cmake_core: 0.9.12 (/opt/ros/foxy/share/ament_cmake_core/cmake)
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
-- Using PYTHON_EXECUTABLE: /usr/bin/python3
-- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
-- Found rosidl_adapter: 1.3.1 (/opt/ros/foxy/share/rosidl_adapter/cmake)
-- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
-- Found std_msgs: 2.0.5 (/opt/ros/foxy/share/std_msgs/cmake)
PROJECTNAME=first_interfaces
msg_files=msg/first_message.msg
interfaces=first_interfacesmsg/first_message.msg
-- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
CMake Error at /opt/ros/foxy/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
  execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
  first_interfaces --arguments-file
  /home/athens/ros2_athens_ws/build/first_interfaces/rosidl_adapter__arguments__first_interfaces.json
  --output-dir
  /home/athens/ros2_athens_ws/build/first_interfaces/rosidl_adapter/first_interfaces
  --output-file
  /home/athens/ros2_athens_ws/build/first_interfaces/rosidl_adapter/first_interfaces.idls)
  returned error code 1:

  Traceback (most recent call last):

    File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
      return _run_code(code, main_globals, None,
    File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
      exec(code, run_globals)
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
      sys.exit(main())
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
      abs_idl_file = convert_to_idl(
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 19, in convert_to_idl
      return convert_msg_to_idl(
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/msg/__init__.py", line 28, in convert_msg_to_idl
      msg = parse_message_string(package_name, input_file.stem, content)
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 540, in parse_message_string
      msg = MessageSpecification(pkg_name, msg_name, fields, constants)
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 387, in __init__
      self.base_type = BaseType(
    File "/opt/ros/foxy/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 204, in __init__
      raise InvalidResourceName(

  rosidl_adapter.parser.InvalidResourceName: 'first_message' is an invalid
  message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'

Call Stack (most recent call first):
  /opt/ros/foxy/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
  CMakeLists.txt:31 (rosidl_generate_interfaces)


-- Configuring incomplete, errors occurred!
See also "/home/athens/ros2_athens_ws/build/first_interfaces/CMakeFiles/CMakeOutput.log".
